doc/nrf/releases_and_maturity/releases/release-notes-2.5.2.rst

```diff
@@ -0,0 +1,90 @@
+.. _ncs_release_notes_252:
+
+|NCS| v2.5.2 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.5.0 <ncs_release_notes_250>` and :ref:`nRF Connect SDK v2.5.1 <ncs_release_notes_251>`.
+
+[TODO]
```

Comment:
Please add below for Wi-Fi section:

```
Wi-Fi
-------
* Fixed an issue with QSPI initialisation on custom board with nRF7002 (SHEL-2372)
```

Reply:
the commit for the fix has a typo, it points to `SHEL-2375`, not sure if we should mention it.

Reply:
> the commit for the fix has a typo, it points to `SHEL-2375`, not sure if we should mention it.

I would say this is not required. Not something the user needs to know.